lib/data/replacements.json

```diff
@@ -670,6 +671,18 @@
       }
     ]
   },
+  "framer-motion-to-motion": {
+    "description": "`framer-motion` and Motion One have merged under the package name `motion`, though `framer-motion` is still being published. `motion` is now the recommended name.",
+    "packageRules": [
+      {
+        "matchCurrentVersion": ">=11.11.12",
+        "matchDatasources": ["npm"],
+        "matchPackageNames": ["framer-motion"],
+        "replacementName": "motion",
+        "replacementVersion": "11.11.12"
```

Comment:
This is not ideal because it could result in a _rollback_ of versions.

I think that while @mattgperry is releasing both packages in parallel, maybe Renovate shouldn't proactively replace `framer-motion` with `motion`. If/once `framer-motion` is stopped then we no longer have a moving target for the replacement

Reply:
I think we should support replace name only. so we can simply leave the replacement version undefined and retain the current version for such use cases

Reply:
OK so requires a modification to the replacement logic to accept empty replacement version?

Reply:
I think so. @zharinov do you know?

Reply:
Seems like so